### ISSUE TITLE
error prevention when filling *long* values

### ISSUE THREAD

**Issue by edouard-lopez** [guillaumevincent/rangevoting#4]
### TL;DR
The UI work fine with single/short value but doesn't scale well with longer values. 

### Issue

For instance with:

> d'une police Chinoise avec la pronociation du caractère, d'une extension dans mon navigateur web, d'une appli mobile, d'une application sur mon PC

![rangevoting-long-values-issue](https://cloud.githubusercontent.com/assets/1212392/9988169/86537278-604e-11e5-9a72-13000a7f4b8a.png)

### Propal

One way would be to replace hint message by a preview of items' value as follow:

![rangevoting-long-values-propal](https://cloud.githubusercontent.com/assets/1212392/9988176/9a7c216e-604e-11e5-9d45-b6c39e0d1753.png)

it gracefully degrade on mobile
![rangevoting-long-values-propal-responsive](https://cloud.githubusercontent.com/assets/1212392/9988201/c377b10a-604e-11e5-86e8-808221c24633.png)

#### Discussion

todo: pro/cons vs. multiple editable fields?
